### PR TITLE
Feature/tvist1 518 hearing post types

### PR DESCRIPTION
**Pull request by jekuaitk**
* Adds two types of hearing posts namely `HearingPostRequest` and `HearingPostResponse`

https://jira.itkdev.dk/browse/TVIST1-518

### CODE REVIEW

File: src/Controller/HearingController.php [hunk 1]
```diff
@@ -73,15 +74,19 @@ public function hearing(CaseEntity $case, HearingPostRepository $hearingPostRepo
 
         // Detect whether most recent hearing post has been forwarded or even exists
         $mostRecentPost = reset($hearingPosts);
-        $hasNewUnforwardedPost = $mostRecentPost && !$mostRecentPost->getForwardedOn();
+        if ($mostRecentPost instanceof HearingPostResponse) {
+            $requiresProcessing = $mostRecentPost && !$mostRecentPost->getApprovedOn();
+        } else {
+            $requiresProcessing = $mostRecentPost && !$mostRecentPost->getForwardedOn();
+        }
```
Comment (rimi-itk) [inline src/Controller/HearingController.php:81]: `$requiresProcessing = … ? … : …;`

File: templates/case/hearing/index.html.twig [hunk 2]
```diff
@@ -91,17 +93,27 @@
                                 <div class="row">
                                     <div class="col">
                                         <h2 class="m-0 p-0">
-                                            {% if not post.getForwardedOn %}
+                                            {% if post.forwardedOn is defined and not post.getForwardedOn %}
+                                                <span class="badge badge-pill badge-warning">N</span>
```
Comment (rimi-itk) [inline templates/case/hearing/index.html.twig:97]: What does “N” stand for? “No”? Should it be translated?

**Review by rimi-itk — CHANGES_REQUESTED** [review]

File: templates/case/hearing/index.html.twig (hunk 2 above)
Comment (jekuaitk) [reply templates/case/hearing/index.html.twig:97]: "N" indicates something new and/or something that needs processing.

File: src/Controller/HearingController.php (hunk 1 above)
Comment (jekuaitk) [reply src/Controller/HearingController.php]: Fixed in ae50d3f482f55c5fb7a2eed4bcc8af9b9388689a.

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**